src/index.ts

```diff
@@ -1805,4 +1806,34 @@ export class MintSDK {
     )
     return data.data
   }
+
+  /**
+   * Returns the information for the walletList includes Event information
+   *
+   * @param walletListId
+   * @returns
+   * walletList: {
+   *  id: string;
+   *  walletListName: string;
+   *  eventSetting: {
+   *    eventImagePreviewUrl: string;
+   *    eventName: string;
+   *    eventDetails: string;
+   *    eventImageId: string;
+   *  }
+   * }
+   *
+   * ```typescript
+   * import ( MintSDK ) from '@kyuzan/mint-sdk-js'
+   * const sdk = new MintSDK(...)
+   * const walletList = await sdk.getWalletList('walletListId)
+   * ```
+   */
+  public getWalletList = async (walletListId: string) => {
```

Comment:
nit: rename to `getWalletListByWalletId`

also update the parameter name to `walletId`